uniffi_bindgen/src/templates/RustString.rs

```diff
@@ -32,6 +33,9 @@ pub unsafe extern "C" fn {{ ci.ffi_string_alloc_from().name() }}(cstr: *const st
 /// The argument *must* be a uniquely-owned pointer previously obtained from a call
 /// into the rust code that returned a string.
 ///
+/// # Safety
+/// In order to free the string, Rust takes ownership of a raw pointer
+/// which is an unsafe operation
```

Comment:
micro-nit: A recent pet peeve of my are `# Safety` comments that point out where the unsafety is coming from, but don't tell you what you need to do to ensure it's safe. Could you please move the preceding sentence about how this must be a uniquely-owned blah-blah inside the safety block?